c7n/resources/s3.py

```diff
@@ -1374,10 +1374,11 @@ def process_bucket(self, bucket):
 class FilterPublicBlock(Filter):
     """Filter for s3 bucket public blocks
 
-    `scope`: Optional: Defaults to Any
-    `enabled`: Optional: Defaults to False
-
-    The defaults essentially check to see if any public blocks are missing
+    If no block args are provided it checks
```

Comment:
your comments seem to have very strange columns on these doc strings, please reflow to 90 column width.